xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -1124,7 +1124,7 @@ bool CGUIMediaWindow::HaveDiscOrConnection(const std::string& strPath, int iDriv
   else if (iDriveType==CMediaSource::SOURCE_TYPE_REMOTE)
   {
     //! @todo Handle not connected to a remote share
-    if ( !g_application.getNetwork().IsConnected() )
+    if ( !CServiceBroker::GetNetwork().IsConnected() )
```

Comment:
please remove spaces if you touch the line

Reply:
done.